content/applications/marketing/survey/scoring.rst

```diff
@@ -0,0 +1,45 @@
+=======
+Scoring
+=======
```

Comment:
I suggested merging the content of this page with the scoring section in the survey essentials rst. I'm not sure how you want to proceed given that we also need to have an intro page (check my main comment on the PR conversation first), but I will skip reviewing this page for now.